src/components/Display.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Display = prop => {
+  const { value } = prop;
+
+  return (
+    <div className="display">
+      <h3>{value}</h3>
+    </div>
+  );
+};
+
+Display.defaultProps = { value: PropTypes.string.isRequired };
```

Comment:
- [ ] The prop `value` inside `Display.defaultProps` should be set to zero.
- [ ] Also the prop type is not defined to string for `value`.
```javascript
// hint
Display.propTypes .....
```

Reply:
implement the requirements